test_cli.py

```diff
@@ -794,6 +798,14 @@ def test(ctx):
     check_contains('::home-', run_command([cl, 'ls', '-w', '/']))
 
 
+@TestModule.register('binary')
+def test(ctx):
+    # Upload a binary file and test it
+    uuid = run_command([cl, 'upload', '/bin/ls'])
+    run_command([cl, 'cat', uuid], binary=True)
```

Comment:
Can we test the output of the cat command to make sure it returns what we expect?

Reply:
Done